Tools/DeployApps/BaasClient.cs

```diff
@@ -406,16 +409,10 @@ public async Task<BaasApp> CreateFlxApp(string name)
                                 write = true,
                             }
                         }
-                    }
+                    },
                 }
             });
 
-            var basicAsymmetricObject = Schemas.GenericFlxBaasRule(Differentiator, "BasicAsymmetricObject");
-            await PostAsync<BsonDocument>($"groups/{_groupId}/apps/{app}/services/{mongoServiceId}/rules", basicAsymmetricObject);
-
-            var allTypesAsymmetricObject = Schemas.GenericFlxBaasRule(Differentiator, "AsymmetricObjectWithAllTypes");
-            await PostAsync<BsonDocument>($"groups/{_groupId}/apps/{app}/services/{mongoServiceId}/rules", allTypesAsymmetricObject);
```

Comment:
I don't remember the exact details either - the reason why I removed the rules is because adding the rules before the schema put the server in a weird state. And it appears like the rules weren't needed because the tests seem to pass without them just fine.

Reply:
That's surprising as I clearly remember adding them for a reason, which I can't recall now. Has something drastically changed on the server side? I expect not, so I'm curious to know now why I needed to add those.

Reply:
Alright, thank you for the [details](https://github.com/realm/realm-dotnet/pull/3096#discussion_r1072375948).